fileio/ft_read_atlas.m

```diff
@@ -1993,6 +1993,15 @@
           fn = strrep(fn, ')', '');
           
           atlas.(fn) = tmp.anatomy(:,:,:,m);
+          if ~isa(atlas.(fn), 'double') && ~isa(atlas.(fn), 'single')
+            % ensure that the probabilistic values are either double or
+            % single precision, do single precision to save memory
+            atlas.(fn) = single(atlas.(fn));
+          end
+          if any(atlas.(fn)(:)>1)
```

Comment:
would it not be better to also explicitly test whether values are below 100? If >1 and >100 I would be confused, and dividing by 100 would not be the best option.

Reply:
good point. I thought about that, but decided for the 'binnenbocht' right now.

but adding a && all(atlas.(fn)(:)<=100) to the if clause would do it, right?